GymBuddies/APIManagers/ParseAPIManager.m

```diff
@@ -44,4 +44,92 @@ +(void)logOut:(ParseManagerLogOutCompletionBlock) completion{
 }
 
 
++(void)fetchBodyZones:(ParseManagerFetchingDataRowsCompletionBlock) completion{
+    PFQuery *query = [PFQuery queryWithClassName:@"BodyZone"];
+    
+    ParseManagerFetchingDataRowsCompletionBlock block = ^void(NSArray *elements, NSError *error){
+        completion(elements, error);
+    };
+    
+    [query findObjectsInBackgroundWithBlock:block];
+}
+
+
++ (Exercise *)createExercise:(Exercise *)exercise
```

Comment:
I would consider renaming this -- since it also accepts another `Exercise` object. It can be a bit unclear to others what this function exactly does. If it is about saving the data then you should explicitly mention something along the line of that